### PR TITLE
Rename sample operators to inspect operators, reimplement sample

### PR DESCRIPTION
**Pull request by blesh**
Just what it says. Relates to #1053

### CODE REVIEW

**Comment by blesh** [comment]
cc/ @staltz @headinthebox @mattpodwysocki

File: spec/operators/sampleTime-spec.js [hunk 1]
```diff
@@ -13,6 +13,26 @@ describe('Observable.prototype.sampleTime', function () {
...
+    var e1 =   hot('----a-^--b----c--------------f----|');
+    var e1subs =         '^                           !';
+    var expected =       '-----------c----------------|';
+    // timer              -----------!----------!---------
+
+    expectObservable(e1.sampleTime(110, rxTestScheduler)).toBe(expected);
+    expectSubscriptions(e1.subscriptions).toBe(e1subs);
+  });
+
+  it('should sample ifnew value has arrived, even if it is the same value', function () {
```
Comment (staltz) [inline spec/operators/sampleTime-spec.js]: small detail, but typo there `ifnew` should be `if new`

File: src/CoreOperators.ts [hunk 2]
```diff
@@ -63,7 +63,6 @@ export interface CoreOperators<T> {
   retry?: (count?: number) => Observable<T>;
   retryWhen?: (notifier: (errors: Observable<any>) => Observable<any>) => Observable<T>;
   sample?: (notifier: Observable<any>) => Observable<T>;
-  sampleTime?: (delay: number, scheduler?: Scheduler) => Observable<T>;
```
Comment (staltz) [inline src/CoreOperators.ts]: What happened to sampleTime? It's not in KitchenSink anymore either.

Comment (blesh) [inline src/CoreOperators.ts]: Oops? haha.

**Comment by blesh** [comment]
merged... thanks for the review @staltz

**Pull request merged**